src/client/Microsoft.Identity.Client/Internal/Logger/IdentityLoggerAdapter.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using Microsoft.Identity.Client.Core;
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.Internal.Logger
+{
+#if !XAMARINMAC20
+    internal class IdentityLoggerAdapter : ILoggerAdapter
+    {
+        private readonly IIdentityLogger _identityLogger;
+        private string _clientInfo;
+        private string _correlationId;
+
+        public bool PiiLoggingEnabled { get; }
+        public bool IsDefaultPlatformLoggingEnabled { get; } = false;
+        public string ClientName { get; }
+        public string ClientVersion { get; }
```

Comment:
Can we include [ApiId](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/5e29d349dde99775bf60f23812e2a2437bb0cca9/src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs#L11) here? This will help understand which flow the log is for.

Reply:
Lets add this in a separate PR. Lots of stuff in here already. Plus I have some questions